static/js/util/rest.js

```diff
@@ -35,6 +35,20 @@ const anySpecificError = code =>
   )
 
 export const any404Error = anySpecificError(404)
-export const any403Error = anySpecificError(403)
 export const anyErrorExcept404 = anyErrorExcept([404])
 export const anyErrorExcept404or410 = anyErrorExcept([404, 410])
+
+// the below code is for our new error_type logic and 401/403 status codes are unreliable indicators
```

Comment:
Suggested edit: `401/403 status codes are unreliable for detecting authentication errors. Our auth endpoints return an error type in the response, and these values can be matched to that error type to properly detect authentication errors.`